ocaml-backend/src/main/java/org/kframework/backend/ocaml/DefinitionToOcaml.java

```diff
@@ -1260,7 +1267,7 @@ public void apply(KApply k) {
                         sb.append("| \"").append(sort.name()).append("\" -> (");
                         KLabel freshFunction = mainModule.freshFunctionFor().apply(sort);
                         encodeStringToFunction(sb, freshFunction);
-                        sb.append(" ([Int counter]) config (-1))\n");
+                        sb.append(" ([Int counter]) config [config] (-1))\n");
```

Comment:
again, I'm not comfortable with us changing the primary code path of execution so substantially just to introduce more error reporting. If you need to construct the stack trace when you reach a stuck configuration, use a series of try blocks.